hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -863,13 +865,35 @@ private void verifyReplication(String volumeName, String bucketName,
     }
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = {"rs-3-3-1024k", "xor-3-5-2048k"})
```

Comment:
You should pass valid values also? probably we can pass Pairs indicating valid config or not. Based valid flag, you can assert differently.